public/js/cg.js

```diff
@@ -140,14 +140,22 @@ app.controller('scoringCtrl', ['$scope', '$interval', '$http', 'socket',
             }
           );
         };
-
+        
         socket.on("score", function (state) {
             $scope.showScore = state.showScore;
             $scope.manualScore = state.manualScore;
+            $scope.showProgress = state.showProgress;
             if(state.manualScore){
               $scope.yorkScore = state.yorkScore;
               $scope.lancScore = state.lancScore;
             };
+            
+            var available = $scope.yorkScore + $scope.lancScore;
+			var toWin = 177.5;
```

Comment:
Could we extract this out to a constant at the top making this easier to change? Might also be worth making note of the total points available as well.